content/zh-cn/docs/reference/access-authn-authz/kubelet-tls-bootstrapping.md

```diff
@@ -744,9 +737,8 @@ To secure these, the kubelet can do one of:
 * create self-signed key and certificate, if a key and certificate are not provided
 * request serving certificates from the cluster server, via the CSR API
 -->
-kubelet 也可以使用**服务（Serving）**证书。kubelet 自身向外提供一个
-HTTPS 末端，包含若干功能特性。要保证这些末端的安全性，kubelet 可以执行以下操作
-之一：
+kubelet 也可以使用**服务（Serving）**证书。kubelet 自身向外提供一个 HTTPS 末端，
```

Comment:
```suggestion
kubelet 也可以使用 **服务（Serving）** 证书。kubelet 自身向外提供一个 HTTPS 末端，
```

Ref: [preview page](https://deploy-preview-34784--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/#client-and-serving-certificates)